pkg/agent/agent_windows.go

```diff
@@ -218,6 +322,71 @@ func (i *Initializer) prepareOVSBridgeOnHNSNetwork() error {
 	return nil
 }
 
+func (i *Initializer) prepareOVSConfigForVM() error {
+	klog.InfoS("Performing OVS configuration", "hostIfName", i.nodeConfig.UplinkNetConfig.Name)
+	hostIfName := i.nodeConfig.UplinkNetConfig.Name
+	uplinkIfName := util.GenUplinkInterfaceName(hostIfName)
+	ovsPorts, ovsErr := i.ovsBridgeClient.GetPortList()
+	if ovsErr != nil {
+		klog.Errorf("Failed to list OVS ports")
+		return ovsErr
+	} else {
+		for index := range ovsPorts {
+			port := &ovsPorts[index]
+			if port.Name == hostIfName {
+				klog.InfoS("Uplink and host configuration exists in OVS")
+				return nil
+			}
+		}
+	}
+
+	success := false
+	uplinkExternalIDs := map[string]interface{}{
```

Comment:
It looks line 343-386 are using the same code as part of function `createOVSPortsAndFlows`, shall we provide a public function in external_node_controller to support creating OVS pair ports? @Anandkumar26 @mengdie-song

Reply:
What is the conclusion for this one? We plan to refactor later?

Reply:
I have not explored it yet. 
For now, I will just add a TODO and remove my Name.

Reply:
Ok